src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -123,6 +123,38 @@ class AggregatorSuite extends SparkSuite {
     p.check()
   }
 
+  @Test def testMax() {
+    val p = Prop.forAll(VariantSampleMatrix.gen(hc, VSMSubgen.random)) { vds =>
```

Comment:
I would say cooking up random VSMs is way overkill to test min/max.  Just test it on some small KeyTables (with nulls, etc.)